ibazel/ibazel.go

```diff
@@ -142,6 +157,7 @@ func (i *IBazel) Cleanup() {
 
```

Comment:
It seems like there should be a step in Cleanup() where the live reload server gets shut down. If not, can you help me understand why if that understanding is wrong?

Reply:
The server runs in a goroutine started with `go i.lrserver.ListenAndServe()`. I'm new to go but from what I've read, when the main process exits all goroutines are stopped where they are and resources are cleaned up. This haven't had any issues with the lrserver port not being released on exit either... although iBazel itself sometimes doesn't quit on CTRL-C and I have to kill the process manually on OSX.

Internally, the lrserver is blocking in the http.Server.Serve() call. Could fork the lrserver and add a signal handler to close the http server and then then iBazel could request a close on Cleanup() and wait for the goroutine to end. That being said, it works fine with go handling the lrserver goroutine cleanup as well.

Reply:
So ended up added a Close() function to lrserver on a fork. It was needed for testing as the Bazel instance is spun up and cleaned up all in the test process so can't rely on go to close down the live reload server goroutine.